client/src/templates/Challenges/redux/create-question-epic.js

```diff
@@ -24,18 +24,22 @@ function filesToMarkdown(challengeFiles = []) {
 
     const fileExtension = challengeFile.ext;
     const fileName = challengeFile.name;
-    const fileType = fileExtension === 'js' ? 'javascript' : fileExtension;
```

Comment:
this is an extra line, the forum highlight javascript with `js` fine, so below I changed `fileType` with `fileExtension`